src/il/comment.nim

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
The file name should be `comments.nim`

Reply:
In my opinion, this file name implies this file has function**s** related to the `Comment` object.

Reply:
ok